pkgs/development/libraries/pmix/default.nix

```diff
@@ -80,6 +81,7 @@ stdenv.mkDerivation (finalAttrs: {
     moveToOutput "bin/pmixcc" "''${!outputDev}"
     moveToOutput "share/pmix/pmixcc-wrapper-data.txt" "''${!outputDev}"
 
+  '' + lib.optionalString (lib.elem "dev" finalAttrs.outputs) ''
```

Comment:
This is a mistake :) It should be conditioned on `stdenv.isLinux`, similarly to how it was done in `openmpi`... It won't change evaluation though.

Reply:
> This is a mistake :) It should be conditioned on `stdenv.isLinux`, similarly to how it was done in `openmpi`... It won't change evaluation though.

Done.